config-model/src/main/java/com/yahoo/searchdefinition/document/SDField.java

```diff
@@ -554,6 +557,17 @@ public void setMatchingType(Matching.Type type) {
         }
     }
 
+    /**
+     * Set the matching type for this field and all subfields.
+     */
+    // TODO: When this is not the same as getMatching().setthis we have a potential for inconsistency. Find the right
```

Comment:
Typo: 'setthis'

Reply:
I just copied the comment from setMatchingType and setMatchingAlgorithm just to ensure that they all got clenaed up when the original author of the comment decided to clean this up. Not sure what he really meant. Hope it was not me......